src/test/ui/generic-associated-types/issue-81712-cyclic-traits.rs

```diff
@@ -0,0 +1,21 @@
+// Regression test for #81712.
+
+#![feature(generic_associated_types)]
+#![allow(incomplete_features)]
+
+trait A {
+    type BType: B<AType = Self>;
+}
+
+trait B {
+    type AType: A<BType = Self>;
+}
+trait C {
```

Comment:
To keep context, I'd leave the comment in here, and change it to past tense.

```suggestion
// rustc did crash on the generic cyclic dependency between traits C and D.
trait C {
```

Reply:
I don't think this is also that helpful. 🤷‍♀️